test/e2e/instrumentation/events.go

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package instrumentation
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	v1beta1 "k8s.io/api/events/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"github.com/onsi/ginkgo"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	eventRetryPeriod  = 1 * time.Second
+	eventRetryTimeout = 1 * time.Minute
+)
+
+var _ = ginkgo.Describe("[sig-instrumentation] Events", func() {
+	f := framework.NewDefaultFramework("events")
+
+	/*
+			   Release : v1.19
+			   Testname: Event resource lifecycle
+			   Description: Create an event, the event MUST exist.
+		           The event is patched with a new message, the check MUST have the update message.
+		           The event is deleted and MUST NOT show up when listing all events.
+	*/
+	ginkgo.It("should ensure that an event can be fetched, patched, deleted, and listed", func() {
+		eventTestName := "event-test"
+		someTime := metav1.MicroTime{Time: time.Unix(1505828956, 0)}
+
+		ginkgo.By("creating a test event")
+		// create a test event in test namespace
```

Comment:
let's create a variable for the client - then promoting to v1 will be much smaller change, i.e.
```
client := f.ClientSet.EventsV1beta1()
```

and use it everywhere below

Reply:
Done.